spec/models/importers/organisation_spec.rb

```diff
@@ -63,4 +63,11 @@
     content_ids = organisation.content_items.pluck(:content_id)
     expect(content_ids).to eq(%w(number-1 number-2))
   end
+
+  it 'raises an exception with an organisation that does not exist' do
```

Comment:
Should change this to `raises an exception with an organisation that does not have any content items`